__tests__/integration/topics.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const db = require("../../db/connection");
+const seed = require("../../db/seeds/seed");
+const testData = require("../../db/data/test-data");
+const { app } = require("../../app");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("/api/topics", () => {
+  test("GET: 200 returns an array of all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics.length).toBe(3);
```

Comment:
Great that you're checking length as well as the contents to avoid false positives if an empty array were passed 👍